src/test/java/org/apache/commons/lang3/ArchUtilsTest.java

```diff
@@ -113,6 +113,10 @@ public void testArchLabels() {
     public void testGetProcessor() {
         assertNotNull(ArchUtils.getProcessor(X86));
         assertNull(ArchUtils.getProcessor("NA"));
+
+        final Processor processor = ArchUtils.getProcessor();
+        assertTrue(processor.isX86());
```

Comment:
This test will fail on machines with other arches (e.g. Apple M1 returns `Processor.Type.AARCH_64`). I suggest we remove it.

cc @garydgregory @arturobernalg